.github/s390x-ci/self-hosted-builder/fs/usr/bin/entrypoint

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Is there a reason not to do it in Python? (My bash foo is bad, @seemethere may be you can give there a hand)

Reply:
I took this part from my colleague as is:
https://github.com/zlib-ng/zlib-ng/blob/develop/arch/s390/self-hosted-builder/fs/usr/bin/entrypoint

No other reason to keep it in bash.